docs/Apps/Screening & Sequencing Data/CRISPR Screening.md

```diff
@@ -29,11 +29,48 @@ CRISPR Screening requires the following two files as input:
 
 **Fastq files**
 
-These are files in fast.gz format. 
+Fastq files stores short read data from high throughput sequencing experiments. The sequence and quality scores are usually put into a single line each.These are files in fast.gz format.
+
+A FASTQ record has the following format:
+
+- A line starting with @, containing the sequence ID.
+
+
+- One or more lines that contain the sequence.
+
+
+- A new line starting with the character +, and being either empty or repeating the sequence ID.
+
+
+- One or more lines that contain the quality scores.
+
+![Fastq file](../../../img/CRISPR/Fastqfile.png) <center>**Figure 2.** Fastq File</center>
+
+The ideal CRISPR sequence length is only ~20 bp nucleotides. If you find your data contains more than ~ 20 nucleotides, there is a high chance there are adapters in the sequence. These experimental adapters need to be trimmed.
+
+![Fastq file showing adapters](../../../img/CRISPR/Fastqfile2.png) <center>**Figure 3.** Fastq File showing adapters</center>
+
+The highlighted sequence is of adapters. We can specify the length of nucleotides to be trimmed in the Trim-5 given under the pipeline and parameter tab.
+
+![Trim-5](../../../img/CRISPR/Trim5.png) <center>**Figure 4.** Trim-5</center> 
 
 **Library file**
 
-This is file in .csv format.
+The library file of CRISPR method stores which sgRNA targets which gene. The file format should be CSV. Each gene associated with multiple sgRNA. 
+
+The file contains three columns:
+
+- **Id**: contains Unique sgRNA id
+
+- **gRNA** **sequence**: Consists of gRNA sequence. The sequences are searched in the fastq files by the CRISPR screening algorithm to calculate the read counts.
+
+- **Gene**: Gene column consists of gene symbol associated with each gRNA sequence
+
```

Comment:
If ID, gRNA sequence and Gene are the exact column names in the library file they should be in italics and not bold.

Reply:
Use consistent indentation as the res of the files. @Shreya-crypto